.github/workflows/validate-new-issue-test.yml

```diff
@@ -105,7 +105,7 @@ jobs:
           issue_number=$(
             basename $(
               gh issue create --title "Greetings, program!" \
-                --body "# Foo\nI don't follow directions."
+                --body "I don't follow directions."
```

Comment:
Nit: The `\n` was coming through verbatim, not as a newline. [E.g.](https://github.com/getsentry/.github-chadwhitacre/issues/416).